modules/charts_month.py

```diff
@@ -0,0 +1,95 @@
+"""
+This module generates charts on console
+"""
+
+from constants import months_list
+from modules.utils import (
+    get_date,
+    get_highest_temperature,
+    get_lowest_temperature,
+    read_data,
+)
+
+
+def get_all_extremes(year_month, path):
```

Comment:
The default value for path? Could it be from constants?

Reply:
done